### ISSUE TITLE
Trackpad scrolling is shit.

### ISSUE THREAD

**Issue by ColtonDRG** [GalliumOS/chromium#1]
Trackpad scrolling does not work well in Chromium for Linux. This is super dumb and we should try to fix it.

**Comment by Houseoftea** [comment]
Is the problem global scrolling or just in chromium?

My first thought was that the trackpad needed to be tweaked, for example to increase sensitivity you would:


Section "InputClass"
    Identifier      "touchpad peppy cyapa"
    MatchIsTouchpad "on"
    MatchDevicePath "/dev/input/event*"
    MatchProduct    "cyapa"
    Option          "FingerLow" "8"
    Option          "FingerHigh" "16"
    Option "SoftButtonAreas" "0% 0 0% 0 0 0 0 0"
    Option "AreaBottomEdge" "0%"
EndSection

But if it is only a chromium issue than this is useless

**Comment by ColtonDRG** [comment]
Nope, the issue is only in Chromium. Most other Linux applications work fine.

**Comment by coolstar** [comment]
this same issue happens in Chromium on Windows too, so if we do get around to fixing it, it might also benefit Windows as well.

(On my C720P with Windows, scrolling works fine in Microsoft Edge, IE and metro apps but is horrible in chrome)

**Comment by ColtonDRG** [comment]
Alright, we'll look into fixing it on Windows too, but Linux should be our priority since that's what we're working with. It should also be noted that this will benefit all Linux (and possibly Windows) users, not just us.

**Comment by Roechamboe** [comment]
There is an option to enable smooth scrolling in "chrome://flags/" I find it to be acceptable, anything further is going to require some reverse engineering that may not be persistent among updates.

**Comment by ColtonDRG** [comment]
Nope. That does make the scrolling smooth, but it actually breaks the functionality worse on the trackpad. It's still not variable. We want scrolling to work the same way it does in ChromiumOS, not kinda similar-ish.
